presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -441,10 +441,10 @@ public synchronized boolean isSubmissionFinished()
          * Retrieves an object that contains all information about the dispatched query.
```

Comment:
The word "Information" seems too generic. What is this information? The query metadata?
Let's try not to be so generic if possible.

Reply:
Changed to
```
Retrieves an object that contains the result of a dispatched query.
```